client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -153,9 +153,7 @@ export const UpgradePlan: React.FunctionComponent< Props > = ( props: Props ) =>
 				</div>
 			) }
 
-			<UpgradePlanDetails isEligibleForTrialPlan={ isEligibleForTrialPlan }>
```

Comment:
This prop was used only to add a class that isn't needed anymore. So we're removing this.